packages/forklift-console-plugin/src/modules/Plans/views/create/steps/GeneralInformationForm.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+import { useForkliftTranslation } from 'src/utils';
+
+import {
+  Form,
+  FormGroup,
+  FormSection,
+  TextInput,
+  Title,
+  ValidatedOptions,
+} from '@patternfly/react-core';
+
+import { FormErrorHelperText } from '../components/FormErrorHelperText';
+
+import { GeneralFormFieldId, getGeneralFormFieldLabels } from './constants';
+
+export const GeneralInformationForm: FC = () => {
+  const { t } = useForkliftTranslation();
+  const {
+    control,
+    formState: { errors },
+  } = useFormContext();
+  const fieldLabels = getGeneralFormFieldLabels(t);
+
+  return (
+    <Form className="pf-v5-u-w-66">
```

Comment:
Since we are planning to migrate soon to PF6, we should try and avoid using the v5 classnames as possible and if needed create our own classnames for easier migration

Reply:
I understand the argument, but these util files are commonly used throughout PF and PF has migration tools that simply find and replace these class names to assist with migrations. sizing class names in particular like this one are just a replacement of "v5" to "v6" in the name, and the styles generally do not change. 

I can add custom css, but it will be more lines in our code we have to then maintain and wonder might conflict with PF styles. That leads to more cost of time if enough custom CSS is added IMO.

Reply:
I did create a css file and re-arranged the Form so that the first FormSection did not require it's custom class. Considering it's just the 1 custom class to make forms 66% the width of the content area, I don't see any harm in either implementation.